tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss

```diff
@@ -39,6 +39,7 @@ section .control-row:not(:last-child) {
 }
 
 .control-name {
+  margin-top: 16px;
```

Comment:
Why the margin top is needed? Isn't the current css setting not enough?

Reply:
For some reason, earlier without this there was no space between the slider and axis drop-down, it must be some unintentional bug from the initial version. Removed.